### DISCUSSION TITLE
SAAS: run a deployment plan over all the tenants excepts the main SaaS one

### DISCUSSION THREAD

**Discussion by PiemP** [OrchardCMS/OrchardCore#9712] [Q&A] [answered]
Hi All, 

It's a long time that I don't align the OrchardCore source with the dev branch and I'm asking if already exist a way to run a deployment plan over all the tenants of an OrchardCore SaaS instance (except on the SaaS tenant).

I hope to be clear on my needs.
Thank you.

**Comment by jptissot** [comment, marked as answer]
I don't think we have that ability yet. 

I did implement it here: https://github.com/StatCan/StatCan.OrchardCore/pull/207/files#diff-5805dbfea15427ad8ab5ead0ff4d0871a7da99ebb61bf316148634da582fb607R60 
Not merged yet as I haven't gotten a chance to properly test this yet.
